docs/static/management/configuring-centralized-pipelines.asciidoc

```diff
@@ -1,8 +1,5 @@
 [[configuring-centralized-pipelines]]
-=== Configuring Centralized Pipeline Management
-++++
-<titleabbrev>Centralized Pipeline Management</titleabbrev>
-++++
```

Comment:
Removing this (at least for now) to avoid having two subsections called "Centralized Pipeline Management" back to back. :-)